### ISSUE TITLE
L003 adding indents between macro & column alias (AS) 

### ISSUE THREAD

**Issue by foundinblank** [sqlfluff/sqlfluff#961]
## Expected Behaviour

We're still trialing SQLFluff and have only one active rule turned on: L003. We added a new field to a series of models, all defined by a macro. See below for example. 

``` sql
SELECT
    *,
    {{ get_utc_offset(x, y) }} AS utc_offset 
FROM tbl  
```

The macro this calls on is defined below, and the indenting of this looks appropriate. 
``` sql
{% macro get_utc_date_offset(x, y) %}

DATEADD(
    y,
    DATEDIFF(TO_DATE(TO_UTC_TIMESTAMP(x, 'America/Los_Angeles')), TO_DATE(x))
)

{% endmacro %}
```

## Observed Behaviour

`sqlfluff lint` throws an `Indentation not consistent with line xx` error. When running `sqlfluff fix` it modifies so that it's adding an indent (two of them in fact, 8 spaces total) between the macro & the `AS` column alias. 

``` sql
SELECT
    *,
    {{ get_utc_offset(x, y) }}        AS utc_offset 
FROM tbl  
```

## Steps to Reproduce

Not sure. All our modified models with the {{ get_utc_offset() }} macro are failing. 

## Version
```
sqlfluff, version 0.5.2
Python 3.7.9
```

## Configuration
```
[sqlfluff]
rules = L003
templater = dbt
# dbt templating does not keep trailing new lines (L009)
exclude_rules = L009
```

## SQLFluff `parse`
```
=== [dbt templater] Compiling dbt project...
[0](1, 1, 1)        |file:
[0](1, 1, 1)        |    statement:
[0](1, 1, 1)        |        with_compound_statement:
[0](1, 1, 1)        |            keyword:                                          'WITH'
[4](1, 1, 5)        |            newline:                                          '\n'
[5](1, 2, 1)        |            newline:                                          '\n'
[6](1, 3, 1)        |            common_table_expression:
[6](1, 3, 1)        |                identifier:                                   'base'
[10](1, 3, 5)       |                whitespace:                                   ' '
[11](1, 3, 6)       |                keyword:                                      'AS'
[13](1, 3, 8)       |                whitespace:                                   ' '
[14](1, 3, 9)       |                start_bracket:                                '('
[15](1, 3, 10)      |                [META] indent:
[15](1, 3, 10)      |                newline:                                      '\n'
[16](1, 4, 1)       |                newline:                                      '\n'
[17](1, 5, 1)       |                whitespace:                                   '    '
[21](1, 5, 5)       |                select_statement:
[21](1, 5, 5)       |                    select_clause:
[21](1, 5, 5)       |                        keyword:                              'SELECT'
[27](1, 5, 11)      |                        [META] indent:
[27](1, 5, 11)      |                        whitespace:                           ' '
[28](1, 5, 12)      |                        select_clause_element:
[28](1, 5, 12)      |                            wildcard_expression:
[28](1, 5, 12)      |                                wildcard_identifier:
[28](1, 5, 12)      |                                    star:                     '*'
[29](1, 5, 13)      |                    whitespace:                               ' '
[30](1, 5, 14)      |                    [META] dedent:
[30](1, 5, 14)      |                    from_clause:
[30](1, 5, 14)      |                        keyword:                              'FROM'
[34](1, 5, 18)      |                        whitespace:                           ' '
[35](1, 5, 19)      |                        from_expression:
[35](1, 5, 19)      |                            [META] indent:
[35](1, 5, 19)      |                            from_expression_element:
[35](1, 5, 19)      |                                table_expression:
[35](1, 5, 19)      |                                    table_reference:
[35](1, 5, 19)      |                                        identifier:           'raw_fivetran_zuora'
[35](1, 5, 19)      |                                        dot:                  '.'
[35](1, 5, 19)      |                                        identifier:           'account'
[61](1, 5, 45)      |                            [META] dedent:
[67](1, 5, 51)      |                newline:                                      '\n'
[68](1, 6, 1)       |                newline:                                      '\n'
[63](1, 7, 1)       |                [META] dedent:
[69](1, 7, 1)       |                end_bracket:                                  ')'
[70](1, 7, 2)       |            comma:                                            ','
[71](1, 7, 3)       |            newline:                                          '\n'
[72](1, 8, 1)       |            newline:                                          '\n'
[73](1, 9, 1)       |            common_table_expression:
[73](1, 9, 1)       |                identifier:                                   'fields'
[79](1, 9, 7)       |                whitespace:                                   ' '
[80](1, 9, 8)       |                keyword:                                      'AS'
[82](1, 9, 10)      |                whitespace:                                   ' '
[83](1, 9, 11)      |                start_bracket:                                '('
[78](1, 9, 12)      |                [META] indent:
[84](1, 9, 12)      |                newline:                                      '\n'
[85](1, 10, 1)      |                newline:                                      '\n'
[86](1, 11, 1)      |                whitespace:                                   '    '
[90](1, 11, 5)      |                select_statement:
[90](1, 11, 5)      |                    select_clause:
[90](1, 11, 5)      |                        keyword:                              'SELECT'
[90](1, 11, 11)     |                        [META] indent:
[96](1, 11, 11)     |                        newline:                              '\n'
[97](1, 12, 1)      |                        whitespace:                           '        '
[105](1, 12, 9)     |                        select_clause_element:
[105](1, 12, 9)     |                            column_reference:
[105](1, 12, 9)     |                                identifier:                   'id'
[107](1, 12, 11)    |                            whitespace:                       ' '
[108](1, 12, 12)    |                            alias_expression:
[108](1, 12, 12)    |                                keyword:                      'AS'
[110](1, 12, 14)    |                                whitespace:                   ' '
[111](1, 12, 15)    |                                identifier:                   'account_id'
[121](1, 12, 25)    |                        comma:                                ','
[122](1, 12, 26)    |                        newline:                              '\n'
[123](1, 13, 1)     |                        whitespace:                           '        '
[131](1, 13, 9)     |                        select_clause_element:
[131](1, 13, 9)     |                            column_reference:
[131](1, 13, 9)     |                                identifier:                   'account_number'
[145](1, 13, 23)    |                        comma:                                ','
[146](1, 13, 24)    |                        newline:                              '\n'
[147](1, 14, 1)     |                        whitespace:                           '        '
[155](1, 14, 9)     |                        select_clause_element:
[155](1, 14, 9)     |                            column_reference:
[155](1, 14, 9)     |                                identifier:                   'additional_email_addresses'
[181](1, 14, 35)    |                        comma:                                ','
[182](1, 14, 36)    |                        newline:                              '\n'
[183](1, 15, 1)     |                        whitespace:                           '        '
[191](1, 15, 9)     |                        select_clause_element:
[191](1, 15, 9)     |                            column_reference:
[191](1, 15, 9)     |                                identifier:                   'allow_invoice_edit'
[209](1, 15, 27)    |                        comma:                                ','
[210](1, 15, 28)    |                        newline:                              '\n'
[211](1, 16, 1)     |                        whitespace:                           '        '
[219](1, 16, 9)     |                        select_clause_element:
[219](1, 16, 9)     |                            column_reference:
[219](1, 16, 9)     |                                identifier:                   'auto_pay'
[227](1, 16, 17)    |                        comma:                                ','
[228](1, 16, 18)    |                        newline:                              '\n'
[229](1, 17, 1)     |                        whitespace:                           '        '
[237](1, 17, 9)     |                        select_clause_element:
[237](1, 17, 9)     |                            column_reference:
[237](1, 17, 9)     |                                identifier:                   'balance'
[244](1, 17, 16)    |                            whitespace:                       ' '
[245](1, 17, 17)    |                            alias_expression:
[245](1, 17, 17)    |                                keyword:                      'AS'
[247](1, 17, 19)    |                                whitespace:                   ' '
[248](1, 17, 20)    |                                identifier:                   'account_current_balance'
[271](1, 17, 43)    |                        comma:                                ','
[272](1, 17, 44)    |                        newline:                              '\n'
[273](1, 18, 1)     |                        whitespace:                           '        '
[281](1, 18, 9)     |                        select_clause_element:
[281](1, 18, 9)     |                            column_reference:
[281](1, 18, 9)     |                                identifier:                   'batch'
[286](1, 18, 14)    |                            whitespace:                       ' '
[287](1, 18, 15)    |                            alias_expression:
[287](1, 18, 15)    |                                keyword:                      'AS'
[289](1, 18, 17)    |                                whitespace:                   ' '
[290](1, 18, 18)    |                                identifier:                   'account_billing_batch_group'
[317](1, 18, 45)    |                        comma:                                ','
[318](1, 18, 46)    |                        newline:                              '\n'
[319](1, 19, 1)     |                        whitespace:                           '        '
[327](1, 19, 9)     |                        select_clause_element:
[327](1, 19, 9)     |                            column_reference:
[327](1, 19, 9)     |                                identifier:                   'bcd_setting_option'
[345](1, 19, 27)    |                        comma:                                ','
[346](1, 19, 28)    |                        newline:                              '\n'
[347](1, 20, 1)     |                        whitespace:                           '        '
[355](1, 20, 9)     |                        select_clause_element:
[355](1, 20, 9)     |                            column_reference:
[355](1, 20, 9)     |                                identifier:                   'bill_cycle_day'
[369](1, 20, 23)    |                        comma:                                ','
[370](1, 20, 24)    |                        newline:                              '\n'
[371](1, 21, 1)     |                        whitespace:                           '        '
[379](1, 21, 9)     |                        select_clause_element:
[379](1, 21, 9)     |                            column_reference:
[379](1, 21, 9)     |                                identifier:                   'bill_to_id'
[389](1, 21, 19)    |                        comma:                                ','
[390](1, 21, 20)    |                        newline:                              '\n'
[391](1, 22, 1)     |                        whitespace:                           '        '
[399](1, 22, 9)     |                        select_clause_element:
[399](1, 22, 9)     |                            column_reference:
[399](1, 22, 9)     |                                identifier:                   'braintree_id_c'
[413](1, 22, 23)    |                        comma:                                ','
[414](1, 22, 24)    |                        newline:                              '\n'
[415](1, 23, 1)     |                        whitespace:                           '        '
[423](1, 23, 9)     |                        select_clause_element:
[423](1, 23, 9)     |                            column_reference:
[423](1, 23, 9)     |                                identifier:                   'created_by_id'
[436](1, 23, 22)    |                            whitespace:                       ' '
[437](1, 23, 23)    |                            alias_expression:
[437](1, 23, 23)    |                                keyword:                      'AS'
[439](1, 23, 25)    |                                whitespace:                   ' '
[440](1, 23, 26)    |                                identifier:                   'account_created_by_id'
[461](1, 23, 47)    |                        comma:                                ','
[462](1, 23, 48)    |                        newline:                              '\n'
[463](1, 24, 1)     |                        whitespace:                           '        '
[471](1, 24, 9)     |                        select_clause_element:
[471](1, 24, 9)     |                            column_reference:
[471](1, 24, 9)     |                                identifier:                   'created_date'
[483](1, 24, 21)    |                            whitespace:                       ' '
[484](1, 24, 22)    |                            alias_expression:
[484](1, 24, 22)    |                                keyword:                      'AS'
[486](1, 24, 24)    |                                whitespace:                   ' '
[487](1, 24, 25)    |                                identifier:                   'account_created_date'
[507](1, 24, 45)    |                        comma:                                ','
[508](1, 24, 46)    |                        newline:                              '\n'
[509](1, 25, 1)     |                        whitespace:                           '        '
[517](1, 25, 9)     |                        select_clause_element:
[517](1, 25, 9)     |                            column_reference:
[517](1, 25, 9)     |                                identifier:                   'credit_balance'
[531](1, 25, 23)    |                        comma:                                ','
[532](1, 25, 24)    |                        newline:                              '\n'
[533](1, 26, 1)     |                        whitespace:                           '        '
[541](1, 26, 9)     |                        select_clause_element:
[541](1, 26, 9)     |                            column_reference:
[541](1, 26, 9)     |                                identifier:                   'crm_id'
[547](1, 26, 15)    |                            whitespace:                       ' '
[548](1, 26, 16)    |                            alias_expression:
[548](1, 26, 16)    |                                keyword:                      'AS'
[550](1, 26, 18)    |                                whitespace:                   ' '
[551](1, 26, 19)    |                                identifier:                   'crm_salesforce_id'
[568](1, 26, 36)    |                        comma:                                ','
[569](1, 26, 37)    |                        newline:                              '\n'
[570](1, 27, 1)     |                        whitespace:                           '        '
[578](1, 27, 9)     |                        select_clause_element:
[578](1, 27, 9)     |                            column_reference:
[578](1, 27, 9)     |                                identifier:                   'customer_service_rep_name'
[603](1, 27, 34)    |                        comma:                                ','
[604](1, 27, 35)    |                        newline:                              '\n'
[605](1, 28, 1)     |                        whitespace:                           '        '
[613](1, 28, 9)     |                        select_clause_element:
[613](1, 28, 9)     |                            column_reference:
[613](1, 28, 9)     |                                identifier:                   'default_payment_method_id'
[638](1, 28, 34)    |                        comma:                                ','
[639](1, 28, 35)    |                        newline:                              '\n'
[640](1, 29, 1)     |                        whitespace:                           '        '
[648](1, 29, 9)     |                        select_clause_element:
[648](1, 29, 9)     |                            column_reference:
[648](1, 29, 9)     |                                identifier:                   'invoice_delivery_prefs_email'
[676](1, 29, 37)    |                        comma:                                ','
[677](1, 29, 38)    |                        newline:                              '\n'
[678](1, 30, 1)     |                        whitespace:                           '        '
[686](1, 30, 9)     |                        select_clause_element:
[686](1, 30, 9)     |                            column_reference:
[686](1, 30, 9)     |                                identifier:                   'invoice_delivery_prefs_print'
[714](1, 30, 37)    |                        comma:                                ','
[715](1, 30, 38)    |                        newline:                              '\n'
[716](1, 31, 1)     |                        whitespace:                           '        '
[724](1, 31, 9)     |                        select_clause_element:
[724](1, 31, 9)     |                            column_reference:
[724](1, 31, 9)     |                                identifier:                   'invoice_template_id'
[743](1, 31, 28)    |                        comma:                                ','
[744](1, 31, 29)    |                        newline:                              '\n'
[745](1, 32, 1)     |                        whitespace:                           '        '
[753](1, 32, 9)     |                        select_clause_element:
[753](1, 32, 9)     |                            column_reference:
[753](1, 32, 9)     |                                identifier:                   'last_invoice_date'
[770](1, 32, 26)    |                        comma:                                ','
[771](1, 32, 27)    |                        newline:                              '\n'
[772](1, 33, 1)     |                        whitespace:                           '        '
[780](1, 33, 9)     |                        select_clause_element:
[780](1, 33, 9)     |                            column_reference:
[780](1, 33, 9)     |                                identifier:                   'mrr'
[783](1, 33, 12)    |                            whitespace:                       ' '
[784](1, 33, 13)    |                            alias_expression:
[784](1, 33, 13)    |                                keyword:                      'AS'
[786](1, 33, 15)    |                                whitespace:                   ' '
[787](1, 33, 16)    |                                identifier:                   'account_mrr'
[798](1, 33, 27)    |                        comma:                                ','
[799](1, 33, 28)    |                        newline:                              '\n'
[800](1, 34, 1)     |                        whitespace:                           '        '
[808](1, 34, 9)     |                        select_clause_element:
[808](1, 34, 9)     |                            column_reference:
[808](1, 34, 9)     |                                identifier:                   'name'
[812](1, 34, 13)    |                            whitespace:                       ' '
[813](1, 34, 14)    |                            alias_expression:
[813](1, 34, 14)    |                                keyword:                      'AS'
[815](1, 34, 16)    |                                whitespace:                   ' '
[816](1, 34, 17)    |                                identifier:                   'account_name'
[828](1, 34, 29)    |                        comma:                                ','
[829](1, 34, 30)    |                        newline:                              '\n'
[830](1, 35, 1)     |                        whitespace:                           '        '
[838](1, 35, 9)     |                        select_clause_element:
[838](1, 35, 9)     |                            column_reference:
[838](1, 35, 9)     |                                identifier:                   'netlify_account_id_c'
[858](1, 35, 29)    |                            whitespace:                       ' '
[859](1, 35, 30)    |                            alias_expression:
[859](1, 35, 30)    |                                keyword:                      'AS'
[861](1, 35, 32)    |                                whitespace:                   ' '
[862](1, 35, 33)    |                                identifier:                   'netlify_account_id'
[880](1, 35, 51)    |                        comma:                                ','
[881](1, 35, 52)    |                        newline:                              '\n'
[882](1, 36, 1)     |                        whitespace:                           '        '
[890](1, 36, 9)     |                        select_clause_element:
[890](1, 36, 9)     |                            column_reference:
[890](1, 36, 9)     |                                identifier:                   'netlify_creator_id_c'
[910](1, 36, 29)    |                            whitespace:                       ' '
[911](1, 36, 30)    |                            alias_expression:
[911](1, 36, 30)    |                                keyword:                      'AS'
[913](1, 36, 32)    |                                whitespace:                   ' '
[914](1, 36, 33)    |                                identifier:                   'netlify_creator_id'
[932](1, 36, 51)    |                        comma:                                ','
[933](1, 36, 52)    |                        newline:                              '\n'
[934](1, 37, 1)     |                        whitespace:                           '        '
[942](1, 37, 9)     |                        select_clause_element:
[942](1, 37, 9)     |                            column_reference:
[942](1, 37, 9)     |                                identifier:                   'notes'
[947](1, 37, 14)    |                            whitespace:                       ' '
[948](1, 37, 15)    |                            alias_expression:
[948](1, 37, 15)    |                                keyword:                      'AS'
[950](1, 37, 17)    |                                whitespace:                   ' '
[951](1, 37, 18)    |                                identifier:                   'account_notes'
[964](1, 37, 31)    |                        comma:                                ','
[965](1, 37, 32)    |                        newline:                              '\n'
[966](1, 38, 1)     |                        whitespace:                           '        '
[974](1, 38, 9)     |                        select_clause_element:
[974](1, 38, 9)     |                            column_reference:
[974](1, 38, 9)     |                                identifier:                   'parent_id'
[983](1, 38, 18)    |                            whitespace:                       ' '
[984](1, 38, 19)    |                            alias_expression:
[984](1, 38, 19)    |                                keyword:                      'AS'
[986](1, 38, 21)    |                                whitespace:                   ' '
[987](1, 38, 22)    |                                identifier:                   'account_parent_account_id'
[1012](1, 38, 47)   |                        comma:                                ','
[1013](1, 38, 48)   |                        newline:                              '\n'
[1014](1, 39, 1)    |                        whitespace:                           '        '
[1022](1, 39, 9)    |                        select_clause_element:
[1022](1, 39, 9)    |                            column_reference:
[1022](1, 39, 9)    |                                identifier:                   'payment_gateway'
[1037](1, 39, 24)   |                        comma:                                ','
[1038](1, 39, 25)   |                        newline:                              '\n'
[1039](1, 40, 1)    |                        whitespace:                           '        '
[1047](1, 40, 9)    |                        select_clause_element:
[1047](1, 40, 9)    |                            column_reference:
[1047](1, 40, 9)    |                                identifier:                   'payment_term'
[1059](1, 40, 21)   |                        comma:                                ','
[1060](1, 40, 22)   |                        newline:                              '\n'
[1061](1, 41, 1)    |                        whitespace:                           '        '
[1069](1, 41, 9)    |                        select_clause_element:
[1069](1, 41, 9)    |                            column_reference:
[1069](1, 41, 9)    |                                identifier:                   'purchase_order_number'
[1090](1, 41, 30)   |                        comma:                                ','
[1091](1, 41, 31)   |                        newline:                              '\n'
[1092](1, 42, 1)    |                        whitespace:                           '        '
[1100](1, 42, 9)    |                        select_clause_element:
[1100](1, 42, 9)    |                            column_reference:
[1100](1, 42, 9)    |                                identifier:                   'sales_rep_name'
[1114](1, 42, 23)   |                        comma:                                ','
[1115](1, 42, 24)   |                        newline:                              '\n'
[1116](1, 43, 1)    |                        whitespace:                           '        '
[1124](1, 43, 9)    |                        select_clause_element:
[1124](1, 43, 9)    |                            column_reference:
[1124](1, 43, 9)    |                                identifier:                   'sequence_set_id'
[1139](1, 43, 24)   |                        comma:                                ','
[1140](1, 43, 25)   |                        newline:                              '\n'
[1141](1, 44, 1)    |                        whitespace:                           '        '
[1149](1, 44, 9)    |                        select_clause_element:
[1149](1, 44, 9)    |                            column_reference:
[1149](1, 44, 9)    |                                identifier:                   'sold_to_id'
[1159](1, 44, 19)   |                        comma:                                ','
[1160](1, 44, 20)   |                        newline:                              '\n'
[1161](1, 45, 1)    |                        whitespace:                           '        '
[1169](1, 45, 9)    |                        select_clause_element:
[1169](1, 45, 9)    |                            column_reference:
[1169](1, 45, 9)    |                                identifier:                   'status'
[1175](1, 45, 15)   |                            whitespace:                       ' '
[1176](1, 45, 16)   |                            alias_expression:
[1176](1, 45, 16)   |                                keyword:                      'AS'
[1178](1, 45, 18)   |                                whitespace:                   ' '
[1179](1, 45, 19)   |                                identifier:                   'account_status'
[1193](1, 45, 33)   |                        comma:                                ','
[1194](1, 45, 34)   |                        newline:                              '\n'
[1195](1, 46, 1)    |                        whitespace:                           '        '
[1203](1, 46, 9)    |                        select_clause_element:
[1203](1, 46, 9)    |                            column_reference:
[1203](1, 46, 9)    |                                identifier:                   'tax_company_code'
[1219](1, 46, 25)   |                        comma:                                ','
[1220](1, 46, 26)   |                        newline:                              '\n'
[1221](1, 47, 1)    |                        whitespace:                           '        '
[1229](1, 47, 9)    |                        select_clause_element:
[1229](1, 47, 9)    |                            column_reference:
[1229](1, 47, 9)    |                                identifier:                   'tax_exempt_certificate_id'
[1254](1, 47, 34)   |                        comma:                                ','
[1255](1, 47, 35)   |                        newline:                              '\n'
[1256](1, 48, 1)    |                        whitespace:                           '        '
[1264](1, 48, 9)    |                        select_clause_element:
[1264](1, 48, 9)    |                            column_reference:
[1264](1, 48, 9)    |                                identifier:                   'tax_exempt_certificate_type'
[1291](1, 48, 36)   |                        comma:                                ','
[1292](1, 48, 37)   |                        newline:                              '\n'
[1293](1, 49, 1)    |                        whitespace:                           '        '
[1301](1, 49, 9)    |                        select_clause_element:
[1301](1, 49, 9)    |                            column_reference:
[1301](1, 49, 9)    |                                identifier:                   'tax_exempt_description'
[1323](1, 49, 31)   |                        comma:                                ','
[1324](1, 49, 32)   |                        newline:                              '\n'
[1325](1, 50, 1)    |                        whitespace:                           '        '
[1333](1, 50, 9)    |                        newline:                              '\n'
[1333](1, 50, 9)    |                        newline:                              '\n'
[1333](1, 50, 9)    |                        select_clause_element:
[1333](1, 50, 9)    |                            function:
[1333](1, 50, 9)    |                                function_name:                'DATEADD'
[1333](1, 50, 9)    |                                start_bracket:                '('
[1337](1, 52, 9)    |                                [META] indent:
[1333](1, 50, 9)    |                                newline:                      '\n'
[1333](1, 50, 9)    |                                whitespace:                   '    '
[1333](1, 50, 9)    |                                expression:
[1333](1, 50, 9)    |                                    column_reference:
[1333](1, 50, 9)    |                                        identifier:           'field_to_convert'
[1333](1, 50, 9)    |                                comma:                        ','
[1333](1, 50, 9)    |                                newline:                      '\n'
[1333](1, 50, 9)    |                                whitespace:                   '    '
[1333](1, 50, 9)    |                                expression:
[1333](1, 50, 9)    |                                    function:
[1333](1, 50, 9)    |                                        function_name:        'DATEDIFF'
[1333](1, 50, 9)    |                                        start_bracket:        '('
[1373](1, 54, 14)   |                                        [META] indent:
[1333](1, 50, 9)    |                                        expression:
[1333](1, 50, 9)    |                                            function:
[1333](1, 50, 9)    |                                                function_name:  'TO_DATE'
[1333](1, 50, 9)    |                                                start_bracket:  '('
[1381](1, 54, 22)   |                                                [META] indent:
[1333](1, 50, 9)    |                                                expression:
[1333](1, 50, 9)    |                                                    function:
[1333](1, 50, 9)    |                                                        function_name:  'TO_UTC_TIMESTAMP'
[1333](1, 50, 9)    |                                                        start_bracket:  '('
[1398](1, 54, 39)   |                                                        [META] indent:
[1333](1, 50, 9)    |                                                        expression:
[1333](1, 50, 9)    |                                                            column_reference:
[1333](1, 50, 9)    |                                                                identifier:  'utc_field'
[1333](1, 50, 9)    |                                                        comma:  ','
[1333](1, 50, 9)    |                                                        whitespace:  ' '
[1333](1, 50, 9)    |                                                        expression:
[1333](1, 50, 9)    |                                                            literal:  "'America/Los_Angeles'"
[1430](1, 54, 71)   |                                                        [META] dedent:
[1333](1, 50, 9)    |                                                        end_bracket:  ')'
[1431](1, 54, 72)   |                                                [META] dedent:
[1333](1, 50, 9)    |                                                end_bracket:  ')'
[1333](1, 50, 9)    |                                        comma:                ','
[1333](1, 50, 9)    |                                        whitespace:           ' '
[1333](1, 50, 9)    |                                        expression:
[1333](1, 50, 9)    |                                            function:
[1333](1, 50, 9)    |                                                function_name:  'TO_DATE'
[1333](1, 50, 9)    |                                                start_bracket:  '('
[1442](1, 54, 83)   |                                                [META] indent:
[1333](1, 50, 9)    |                                                expression:
[1333](1, 50, 9)    |                                                    column_reference:
[1333](1, 50, 9)    |                                                        identifier:  'utc_field'
[1451](1, 54, 92)   |                                                [META] dedent:
[1333](1, 50, 9)    |                                                end_bracket:  ')'
[1452](1, 54, 93)   |                                        [META] dedent:
[1333](1, 50, 9)    |                                        end_bracket:          ')'
[1333](1, 50, 9)    |                                newline:                      '\n'
[1454](1, 55, 1)    |                                [META] dedent:
[1333](1, 50, 9)    |                                end_bracket:                  ')'
[1333](1, 50, 9)    |                            newline:                          '\n'
[1333](1, 50, 9)    |                            newline:                          '\n'
[1403](1, 50, 79)   |                            whitespace:                       ' '
[1404](1, 50, 80)   |                            alias_expression:
[1404](1, 50, 80)   |                                keyword:                      'AS'
[1406](1, 50, 82)   |                                whitespace:                   ' '
[1407](1, 50, 83)   |                                identifier:                   'tax_exempt_effective_date'
[1432](1, 50, 108)  |                        comma:                                ','
[1433](1, 50, 109)  |                        newline:                              '\n'
[1434](1, 51, 1)    |                        whitespace:                           '        '
[1442](1, 51, 9)    |                        select_clause_element:
[1442](1, 51, 9)    |                            column_reference:
[1442](1, 51, 9)    |                                identifier:                   'tax_exempt_entity_use_code'
[1468](1, 51, 35)   |                        comma:                                ','
[1469](1, 51, 36)   |                        newline:                              '\n'
[1470](1, 52, 1)    |                        whitespace:                           '        '
[1478](1, 52, 9)    |                        select_clause_element:
[1478](1, 52, 9)    |                            column_reference:
[1478](1, 52, 9)    |                                identifier:                   'tax_exempt_expiration_date'
[1504](1, 52, 35)   |                        comma:                                ','
[1505](1, 52, 36)   |                        newline:                              '\n'
[1506](1, 53, 1)    |                        whitespace:                           '        '
[1514](1, 53, 9)    |                        select_clause_element:
[1514](1, 53, 9)    |                            column_reference:
[1514](1, 53, 9)    |                                identifier:                   'tax_exempt_issuing_jurisdiction'
[1545](1, 53, 40)   |                        comma:                                ','
[1546](1, 53, 41)   |                        newline:                              '\n'
[1547](1, 54, 1)    |                        whitespace:                           '        '
[1555](1, 54, 9)    |                        select_clause_element:
[1555](1, 54, 9)    |                            column_reference:
[1555](1, 54, 9)    |                                identifier:                   'tax_exempt_status'
[1572](1, 54, 26)   |                        comma:                                ','
[1573](1, 54, 27)   |                        newline:                              '\n'
[1574](1, 55, 1)    |                        whitespace:                           '        '
[1582](1, 55, 9)    |                        select_clause_element:
[1582](1, 55, 9)    |                            column_reference:
[1582](1, 55, 9)    |                                identifier:                   'total_debit_memo_balance'
[1606](1, 55, 33)   |                        comma:                                ','
[1607](1, 55, 34)   |                        newline:                              '\n'
[1608](1, 56, 1)    |                        whitespace:                           '        '
[1616](1, 56, 9)    |                        select_clause_element:
[1616](1, 56, 9)    |                            column_reference:
[1616](1, 56, 9)    |                                identifier:                   'unapplied_balance'
[1633](1, 56, 26)   |                        comma:                                ','
[1634](1, 56, 27)   |                        newline:                              '\n'
[1635](1, 57, 1)    |                        whitespace:                           '        '
[1643](1, 57, 9)    |                        select_clause_element:
[1643](1, 57, 9)    |                            column_reference:
[1643](1, 57, 9)    |                                identifier:                   'unapplied_credit_memo_amount'
[1671](1, 57, 37)   |                        comma:                                ','
[1672](1, 57, 38)   |                        newline:                              '\n'
[1673](1, 58, 1)    |                        whitespace:                           '        '
[1681](1, 58, 9)    |                        select_clause_element:
[1681](1, 58, 9)    |                            column_reference:
[1681](1, 58, 9)    |                                identifier:                   'updated_by_id'
[1694](1, 58, 22)   |                            whitespace:                       ' '
[1695](1, 58, 23)   |                            alias_expression:
[1695](1, 58, 23)   |                                keyword:                      'AS'
[1697](1, 58, 25)   |                                whitespace:                   ' '
[1698](1, 58, 26)   |                                identifier:                   'account_updated_by_id'
[1719](1, 58, 47)   |                        comma:                                ','
[1720](1, 58, 48)   |                        newline:                              '\n'
[1721](1, 59, 1)    |                        whitespace:                           '        '
[1729](1, 59, 9)    |                        select_clause_element:
[1729](1, 59, 9)    |                            column_reference:
[1729](1, 59, 9)    |                                identifier:                   'updated_date'
[1741](1, 59, 21)   |                            whitespace:                       ' '
[1742](1, 59, 22)   |                            alias_expression:
[1742](1, 59, 22)   |                                keyword:                      'AS'
[1744](1, 59, 24)   |                                whitespace:                   ' '
[1745](1, 59, 25)   |                                identifier:                   'account_last_updated_date'
[1770](1, 59, 50)   |                        comma:                                ','
[1771](1, 59, 51)   |                        newline:                              '\n'
[1772](1, 60, 1)    |                        whitespace:                           '        '
[1780](1, 60, 9)    |                        select_clause_element:
[1780](1, 60, 9)    |                            column_reference:
[1780](1, 60, 9)    |                                identifier:                   'vatid'
[1785](1, 60, 14)   |                        comma:                                ','
[1786](1, 60, 15)   |                        newline:                              '\n'
[1787](1, 61, 1)    |                        whitespace:                           '        '
[1795](1, 61, 9)    |                        select_clause_element:
[1795](1, 61, 9)    |                            column_reference:
[1795](1, 61, 9)    |                                identifier:                   'sold_to_contact_id'
[1813](1, 61, 27)   |                        comma:                                ','
[1814](1, 61, 28)   |                        newline:                              '\n'
[1815](1, 62, 1)    |                        whitespace:                           '        '
[1823](1, 62, 9)    |                        select_clause_element:
[1823](1, 62, 9)    |                            column_reference:
[1823](1, 62, 9)    |                                identifier:                   'bill_to_contact_id'
[1841](1, 62, 27)   |                    newline:                                  '\n'
[1842](1, 63, 1)    |                    newline:                                  '\n'
[1843](1, 64, 1)    |                    whitespace:                               '    '
[1901](1, 71, 5)    |                    [META] dedent:
[1847](1, 64, 5)    |                    from_clause:
[1847](1, 64, 5)    |                        keyword:                              'FROM'
[1851](1, 64, 9)    |                        whitespace:                           ' '
[1852](1, 64, 10)   |                        from_expression:
[1906](1, 71, 10)   |                            [META] indent:
[1852](1, 64, 10)   |                            from_expression_element:
[1852](1, 64, 10)   |                                table_expression:
[1852](1, 64, 10)   |                                    table_reference:
[1852](1, 64, 10)   |                                        identifier:           'base'
[1910](1, 71, 14)   |                            [META] dedent:
[1856](1, 64, 14)   |                    newline:                                  '\n'
[1857](1, 65, 1)    |                    whitespace:                               '    '
[1861](1, 65, 5)    |                    where_clause:
[1861](1, 65, 5)    |                        keyword:                              'WHERE'
[1920](1, 72, 10)   |                        [META] indent:
[1866](1, 65, 10)   |                        whitespace:                           ' '
[1867](1, 65, 11)   |                        expression:
[1867](1, 65, 11)   |                            keyword:                          'NOT'
[1870](1, 65, 14)   |                            whitespace:                       ' '
[1871](1, 65, 15)   |                            column_reference:
[1871](1, 65, 15)   |                                identifier:                   '_fivetran_deleted'
[1942](1, 72, 32)   |                        [META] dedent:
[1888](1, 65, 32)   |                newline:                                      '\n'
[1889](1, 66, 1)    |                newline:                                      '\n'
[1944](1, 74, 1)    |                [META] dedent:
[1890](1, 67, 1)    |                end_bracket:                                  ')'
[1891](1, 67, 2)    |            newline:                                          '\n'
[1892](1, 68, 1)    |            newline:                                          '\n'
[1893](1, 69, 1)    |            select_statement:
[1893](1, 69, 1)    |                select_clause:
[1893](1, 69, 1)    |                    keyword:                                  'SELECT'
[1953](1, 76, 7)    |                    [META] indent:
[1899](1, 69, 7)    |                    whitespace:                               ' '
[1900](1, 69, 8)    |                    select_clause_element:
[1900](1, 69, 8)    |                        wildcard_expression:
[1900](1, 69, 8)    |                            wildcard_identifier:
[1900](1, 69, 8)    |                                star:                         '*'
[1901](1, 69, 9)    |                whitespace:                                   ' '
[1956](1, 76, 10)   |                [META] dedent:
[1902](1, 69, 10)   |                from_clause:
[1902](1, 69, 10)   |                    keyword:                                  'FROM'
[1906](1, 69, 14)   |                    whitespace:                               ' '
[1907](1, 69, 15)   |                    from_expression:
[1961](1, 76, 15)   |                        [META] indent:
[1907](1, 69, 15)   |                        from_expression_element:
[1907](1, 69, 15)   |                            table_expression:
[1907](1, 69, 15)   |                                table_reference:
[1907](1, 69, 15)   |                                    identifier:               'fields'
[1967](1, 76, 21)   |                        [META] dedent:
```

**Comment by barrywhart** [comment]
Can you provide a dbt project file and brief instructions how to reproduce the issue? When I use your SQL file and `.sqlfluff` file, I get the following error:

```
$ sqlfluff fix test.sql
dbt.exceptions.DbtProjectError: Runtime Error
  no dbt_project.yml found at expected path /Users/bhart/dev/sqlfluff/dbt_project.yml
```

**Comment by pwildenhain** [comment]
@barrywhart Run `pip install dbt` to install the package. 
Then set up a "profile" which connects to a database (it can be any database, for example bigquery or postgres work out of the box with `dbt`): https://docs.getdbt.com/dbt-cli/configure-your-profile

Then run `dbt init` to create a new `dbt` project.

The connect it to your profile you created in the `dbt_project.yml` file.

See if everything is working with `dbt debug`.

Trial a compilation with `dbt compile`. Files can be found in the `target/compiled/` directory

For this example specifically, your macros will be saved in a `macros` directory.

That should be enough to get started, holler on slack if things aren't going right.

**Comment by barrywhart** [comment]
@pwildenhain: These instructions have been really helpful, thanks! I set up a project, and I'm about ready to try and reproduce the issue.

**Comment by barrywhart** [comment]
I tested this on the latest code in `main`. While I wasn't able to get it running in dbt, I ran it using the jinja templater. I'm not seeing any lint warnings. Perhaps this was fixed already as part of some other, similar issue.
